pandas/io/formats/style.py

```diff
@@ -1308,63 +1308,78 @@ def background_gradient(
 
         Parameters
         ----------
         cmap : str or colormap
             Matplotlib colormap.
         low : float
-            Compress the range by the low.
+            Compress the color range at the low end. This is a multiple of the data
+            range to extend below the minimum; sound values usually in [0, 0.5],
+            defaults to 0.
         high : float
-            Compress the range by the high.
+            Compress the color range at the high end. This is a multiple of the data
+            range to extend above the maximum; sound values usually in [0, 0.5],
+            defaults to 0.
         axis : {0 or 'index', 1 or 'columns', None}, default 0
             Apply to each column (``axis=0`` or ``'index'``), to each row
             (``axis=1`` or ``'columns'``), or to the entire DataFrame at once
             with ``axis=None``.
         subset : IndexSlice
             A valid slice for ``data`` to limit the style application to.
         text_color_threshold : float or int
-            Luminance threshold for determining text color. Facilitates text
-            visibility across varying background colors. From 0 to 1.
-            0 = all text is dark colored, 1 = all text is light colored.
+            Luminance threshold for determining text color in [0, 1]. Facilitates text
+            visibility across varying background colors. All text is dark if 0, and
+            light if 1, defaults to 0.408.
 
             .. versionadded:: 0.24.0
 
         vmin : float, optional
             Minimum data value that corresponds to colormap minimum value.
-            When None (default): the minimum value of the data will be used.
+            If not specified the minimum value of the data will be used.
 
             .. versionadded:: 1.0.0
 
         vmax : float, optional
             Maximum data value that corresponds to colormap maximum value.
-            When None (default): the maximum value of the data will be used.
+            If not specified the maximum value of the data will be used.
 
             .. versionadded:: 1.0.0
 
+        gmap : array-like, optional
+            Gradient map for determining the background colors. If not supplied
+            will use the input data from rows, columns or frame. Must be an
+            identical shape for sampling columns, rows or DataFrame based on ``axis``.
+
+            .. versionadded:: 1.3.0
+
         Returns
         -------
         self : Styler
 
-        Raises
-        ------
-        ValueError
-            If ``text_color_threshold`` is not a value from 0 to 1.
-
         Notes
         -----
-        Set ``text_color_threshold`` or tune ``low`` and ``high`` to keep the
-        text legible by not using the entire range of the color map. The range
-        of the data is extended by ``low * (x.max() - x.min())`` and ``high *
-        (x.max() - x.min())`` before normalizing.
+        When using ``low`` and ``high`` the range
```

Comment:
can you add some examples

Reply:
can I use images? difficult to convey the effect of these examples without visual representation

Reply:
![Screen Shot 2021-02-22 at 16 23 55](https://user-images.githubusercontent.com/24256554/108729135-837cc300-752a-11eb-852d-91d59fbccbe5.png)
![Screen Shot 2021-02-22 at 16 24 04](https://user-images.githubusercontent.com/24256554/108729154-85df1d00-752a-11eb-8692-993c48bed0b7.png)

Reply:
I *think* you can use images. see the mpl examples for this (if not sphinx docs).....